Tools/WinMLRunner/src/Run.cpp

```diff
@@ -291,18 +291,12 @@ HRESULT CreateSession(LearningModelSession& session, IDirect3DDevice& winrtDevic
                      // the selectedAdapter
                      std::cout << "Using DXGI for adapter creation.." << std::endl;
                      LUID adapterLuid;
-                     THROW_IF_FAILED(spAdapter->GetLUID(&adapterLuid));
+                     THROW_IF_FAILED(spAdapter->GetProperty(DXCoreAdapterProperty::InstanceLuid, &adapterLuid));
                      THROW_IF_FAILED(dxgiFactory4->EnumAdapterByLuid(adapterLuid, __uuidof(IDXGIAdapter),
                                                                      spDxgiAdapter.put_void()));
                      pAdapter = spDxgiAdapter.get();
                  }
              }
-             else
-             {
-                 // Need to enable experimental features to create D3D12 Device with adapter that has compute only
-                 // capabilities.
-                 THROW_IF_FAILED(D3D12EnableExperimentalFeatures(1, &D3D12ComputeOnlyDevices, nullptr, 0));
```

Comment:
>D3D12EnableExperimentalFeatures [](start = 33, length = 31)

why did we remove this one?

Reply:
"Simon has done the work, It is no longer needed and we can use MCDM devices without enabling Developer Mode.
D3D12EnableExperimentalFeatures() fails if Developer Mode is not enabled and that causes WinMLRunner.exe to exit."